### DISCUSSION TITLE
find two bugs

### DISCUSSION THREAD

**Discussion by MansonLuo** [tyron12233/CodeAssist#467] [Q&A]
1.  when i create projects, i keep application name and package name empty, because i think those two field has default value(My Application, com.mycompany.myapp).the application create successfully, but it faild when i build it, the error message is package name cannot be empty, WTF??!!

2.  so i decide to delete this project. i long press project and click delete button meantime  
, the main.xml file is open at editor.
guess what?  the ide app itself crash poping a red error screen!!

**Comment by ToolDroidYT** [comment]
> 1. when i create projects, i keep application name and package name empty, because i think those two field has default value(My Application, com.mycompany.myapp).the application create successfully, but it faild when i build it, the error message is package name cannot be empty, WTF??!!
> 2. so i decide to delete this project. i long press project and click delete button meantime
>    , the main.xml file is open at editor.
>    guess what?  the ide app itself crash poping a red error screen!!

Ok 👍
